stats/stats.go

```diff
@@ -135,11 +135,13 @@ var (
 		2000, 3000, 4000, 5000, 7500, 10000, // another order of magnitude coarsly covered
 		20000, 30000, 40000, 50000, 75000, 100000, // ditto, the end
 	}
-	numValues  = len(histogramBucketValues)
-	numBuckets = numValues + 1 // 1 special first bucket is <= 0; and 1 extra last bucket is > 100000
-	firstValue = float64(histogramBucketValues[0])
-	lastValue  = float64(histogramBucketValues[numValues-1])
-	val2Bucket []int
+	numValues            = len(histogramBucketValues)
+	numBuckets           = numValues + 1 // 1 special first bucket is <= 0; and 1 extra last bucket is > 100000
+	firstValue           = float64(histogramBucketValues[0])
+	lastValue            = float64(histogramBucketValues[numValues-1])
+	maxArrayValue        = int32(1000) // Last value looked up as O(1) array, the rest is linear search
+	val2Bucket           []int         //up to 1000
```

Comment:
cosmetic/diff minimization: move that line above the maxArrayValue and insert a blank line and re-go-fmt - tia (the unchanged block won't look changed as it shows now just because of whitespace; I wish github would ignore whitespace for go files...)

Reply:
ping ^^^